.storybook/main.ts

```diff
@@ -10,12 +10,13 @@ const storybookConfig: StorybookConfig = {
     '@storybook/addon-a11y',
     '@storybook/addon-storyshots',
     '@storybook/addon-storyshots-puppeteer',
-    'storybook-addon-designs',
+    '@storybook/addon-designs',
   ],
   docs: {
     autodocs: true,
   },
   framework: {
+    // NextJS uses webpack internally, so we want to match that environment as close as possible
     name: '@storybook/react-webpack5',
     options: {},
```

Comment:
That's sad. I wanted to switch amino to `@storybook/react-vite` couple weeks ago. I have `zonos-elements` storybook setup with `vite`, and run command `p dev` took only 3 seconds.